redirects.js

```diff
@@ -0,0 +1,53 @@
+// We need to maintain custom redirects as it's not handled automatically by Docusaurus: https://github.com/facebook/docusaurus/issues/3407
```

Comment:
Can we also support
- `/docs` -> `/docs/installation`
- `/community` -> `/community/installation`
?

Later it would be good to support
- `/docs/{version}` -> `/docs/{version}/installation` - but I would keep that for later 🙂 Thanks!

Reply:
I will try to do that (/docs -> /docs/installation, /community -> /community/installation)

Let's leave versioning  when we will have at least 2 versions + next